data/autoyast_sle15/autoyast_firstboot.xml

```diff
@@ -95,4 +118,32 @@
         <timeout config:type="integer">0</timeout>
       </yesno_messages>
     </report>
+    <scripts>
+        <chroot-scripts config:type="list">
+            <script>
+            <chrooted config:type="boolean">true</chrooted>
+                <source>
+                    <![CDATA[#!/bin/bash
+                    for i in YUI_HTTP_PORT={{YUI_PORT}} YUI_HTTP_REMOTE=1 YUI_REUSE_PORT=1 Y2DEBUG=1; do
+                    echo "export $i" >> /usr/lib/YaST2/startup/Firstboot-Stage/S01-rest-api;
+                    done
+                    chmod +x /usr/lib/YaST2/startup/Firstboot-Stage/S01-rest-api;
+                    ]]>
+                </source>
+            </script>
+        </chroot-scripts>
+        <post-scripts config:type="list">
+            <script>
+            <filename>install_libyui.sh</filename>
+            <interpreter>shell</interpreter>
+            <location/>
+            <feedback config:type="boolean">false</feedback>
+            <source><![CDATA[#!/bin/sh
```

Comment:
wondering why don't install it as a package instead of in a script, if there is a reason, we could document it or write it in a separate the commit (as I don't know why you need the .pid)

Reply:
Package is named "libyui-rest-api[version]" which I think is a bad choice, as for example here it cannot be installed with autoyast without that [version] number. When installed with zypper, it works, as it gets installed by "capability" instead of by name, but this cannot be done with the profile.
for the .pid, see Q9 here : https://documentation.suse.com/de-de/sles/15-SP1/html/SLES-all/app-ay-faq.html